### PR TITLE
magento/merchdocs#1102: UPS. Access License Number, clarification

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

This pull request (PR)describes changing the 'Access License Number' to the 'Access Key' title

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [x] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/shipping/ups.html

### CODE REVIEW

File: src/shipping/ups.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ To offer this shipping method to your customers, you must first open an account
 
 1. For a UPS XML account (default), set **UPS Type** to `United Parcel Service XML`. Then do the following:
 
-   - Enter the following UPS credentials: **User ID**, **Access License Number**, and **Password**
+   - Enter the following UPS credentials: **User ID**, **Access Key** (must be 16 symbols), and **Password**
```
Comment (atwixfirster) [inline src/shipping/ups.md:28]: Thank you for your contributions, @ZhovnirMaria !

Please note, that `Access License Number` is correct Magento label. Could you please make a small editorial fix in your addition:

```
    - Enter the following UPS credentials: **User ID**, **Access License Number** (UPS `Access Key`, should be 16 symbols value), and **Password**
```

**Review by atwixfirster — CHANGES_REQUESTED** [review]

File: src/shipping/ups.md (hunk 1 above)
Comment (ZhovnirMaria) [reply src/shipping/ups.md]: Hi, @atwixfirster 
I made some changes to the request, you can check them.
Thank you!

**Review by atwixfirster — APPROVED** [review]
Thank you for your contributions, @ZhovnirMaria 

Approve from my side ✅ 

CC: @dobooth

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import code to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@dobooth the branch with code successfully imported into`magento-devdocs/merchdocs` repository. Branch name: `imported-magento-merchdocs-1106`.

**Pull request merged**